service/src/main/java/bio/terra/workspace/db/model/DbUpdater.java

```diff
@@ -0,0 +1,204 @@
+package bio.terra.workspace.db.model;
+
+import bio.terra.workspace.db.DbSerDes;
+import bio.terra.workspace.service.resource.model.CloningInstructions;
+import bio.terra.workspace.service.resource.model.CommonUpdateParameters;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.annotation.Nullable;
+import org.apache.commons.lang3.StringUtils;
+
+public class DbUpdater {
+  public static class UpdateData {
+    private String name;
+    private String description;
+    private String dbCloningInstructions;
+    private String jsonAttributes;
+
+    public @Nullable String getName() {
+      return name;
+    }
+
+    public UpdateData setName(@Nullable String name) {
+      this.name = name;
+      return this;
+    }
+
+    public @Nullable String getDescription() {
+      return description;
+    }
+
+    public UpdateData setDescription(@Nullable String description) {
+      this.description = description;
+      return this;
+    }
+
+    @JsonIgnore
```

Comment:
Instead of cloning the enum classes, I took the approach of only cloning the base string data in the static class. So the data that is serialized is `dbCloningInstructions`.